README.md

```diff
@@ -96,30 +101,42 @@ tsunkua = elotl.corpus.load('tsunkua')
       print(row[3]) # document
 ```
 
-```bash
+```
 Una vez una señora se emborrachó
 nándi na ra t'u̱xú bintí
 Otomí del Estado de México (ots)
 El otomí de toluca, Yolanda Lastra
-
 ```
 
 
 ## Package structure
 
-The following structure is a reference. As the package grows it will be better documented.
-
-```
-elotl/                              Top-level package
-          __init__.py               Initialize the package
-          corpora/                  Here are the corpus data
-          corpus/                   Subpackage to load corpus     
-          nahuatl/                  Nahuatl language subpackage
-                  orthography.py    Module to normalyze nahuatl orthography and phonemas
-          utils/                    Subpackage with useful functions and files
-                  fst/              Finite State Transducer functions
-                        att/        Module with static .att files
-test/                               Unit test scripts
+The following structure is a reference. As the package grows it will be better
+documented.
+
+```
+├── CODE_OF_CONDUCT.md
+├── CONTRIBUTING.md
+├── dist
+├── docs
+├── elotl                           Top-level package
+    ├── corpora                     Here are the corpus data
+    ├── corpus                      Subpackage to load corpus
+    ├── huave
+    ├── __init__.py                 Initialize the package
+    ├── nahuatl                     Nahuatl language subpackage
+        └── orthography.py          Module to normalyze nahuatl orthography and phonemas
+    ├── otomi
```

Comment:
Feel free to update missing desc

Reply:
I think this is mostly correct. The only thing (I think) missing is that huave, nahuatl, and otomi all have a `morphology.py` Module for morphological analysis.